lib/time_difference.rb

```diff
@@ -56,9 +56,11 @@ def in_general
     end]
   end
 
-  def humanize
+  def humanize(only_parts = TIME_COMPONENTS)
```

Comment:
unfortunately no can do. 😢 

The gem supports ruby version 1.9, introducing named params would break that.